src/makeCalculator.js

```diff
@@ -36,8 +36,36 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      this.result /= num;
+    },
+    operate(callback, num) {
+      callback.call(this, num);
```

Comment:
```suggestion
      callback(num);
```
Just call it like that

Reply:
yes, we have to use `call`, otherwise the call context will be lost
you've got it the right way